typhos/tools/console.py

```diff
@@ -186,7 +186,7 @@ def _add_device(self, device):
             script = utils.code_from_device(device)
             self.execute(script)
         except Exception:
-            logger.exception("Unable to add device %r to TyphosConsole.",
+            logger.error("Unable to add device %r to TyphosConsole.",
                              device.name)
```

Comment:
The continuous integration checks [here](https://app.travis-ci.com/github/pcdshub/typhos/jobs/570066998) indicate:
```
typhos/tools/console.py:190:30: E127 continuation line over-indented for visual indent
```
This means it's a style issue, saying that the indentation from the change isn't good.
An example of an acceptable indentation style would be:
```python
logger.error(
    "Log message %s",
    argument
)
```